Phonebook.Frontend/src/app/modules/table/dialogs/table-settings-dialog/table-settings.dialog.html

```diff
@@ -61,3 +62,6 @@ <h2 i18n="TableSettingsDialog|Title of displayed table columns@@TableSettingsDia
     </div>
   </div>
 </mat-dialog-content>
+<mat-dialog-actions style="justify-content: flex-end;">
+  <button mat-stroked-button mat-icon-button color="primary" mat-dialog-close><mat-icon>close</mat-icon></button>
```

Comment:
There can only be one directive applied to the button, either `mat-stroked-button` or `mat-icon-button`